bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -40,8 +37,6 @@ pub const LOG_TARGET: &str = "snowbridge-system-frontend";
 
 #[derive(Encode, Decode, Debug, PartialEq, Clone, TypeInfo)]
 pub enum EthereumSystemCall {
-	#[codec(index = 1)]
-	CreateAgent { location: Box<VersionedLocation>, fee: u128 },
 	#[codec(index = 2)]
```

Comment:
Index should be changed right?

```suggestion
	#[codec(index = 1)]
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/10/commits/7f65b82a2a65cf2a5f92171db77c3fc2dcd28bc3
Though prefer to start from zero.